libs/barista-components/experimental/combobox/src/combobox.ts

```diff
@@ -90,6 +90,10 @@ export class DtComboboxFilterChange {
   constructor(public filter: string, public isResetEvent: boolean = false) {}
 }
 
+// We need to save the instance of the combobox that currently
+// has a flap open, to make sure we can close the old one.
+let currentlyOpenCombobox: DtCombobox<any> | null = null;
```

Comment:
NIT: maybe be less explicit

```suggestion
let currentlyOpenCombobox: DtCombobox<unknown> | null = null;
```